### PR TITLE
Segfault in debug menu fix

### PR DESCRIPTION
**Pull request by TheQuantumPhysicist**
A segmentation fault was caused by accessing vector elements that may not exist. A condition that tests the size of the vector is added now.

### CODE REVIEW

File: src/rpcwallet.cpp [hunk 1]
```diff
@@ -1429,10 +1429,12 @@ Value walletpassphrase(const Array& params, bool fHelp)
             "walletpassphrase <passphrase> <timeout> [stakingonly]\n"
             "Stores the wallet decryption key in memory for <timeout> seconds.\n"
             "if [stakingonly] is true sending functions are disabled.");
-    if (params[1].get_int64() > 99999999 || params[1].get_int64() < 0) {
-    	throw runtime_error(
-            "walletpassphrase <passphrase> <timeout> [stakingonly]\n"
-            "Maximum timeout value is 99999999 use timeout of 0 to never re-lock, negative values not allowed");
+    if(params.size() > 2) {
```
Comment (NeblioTeam) [inline src/rpcwallet.cpp]: should this be >= 2 ??

Comment (TheQuantumPhysicist) [reply src/rpcwallet.cpp]: Yes, because element [0] is 1, and [1] is 2.

Comment (TheQuantumPhysicist) [reply src/rpcwallet.cpp]: Yes, but should be `>=`. I'll fix it.

Comment (TheQuantumPhysicist) [reply src/rpcwallet.cpp]: I couldn't reproduce an error for the second array access. But it's definitely good practice to test it before accessing it. I'll do that

**Pull request merged**